app/models/user.rb

```diff
@@ -34,6 +34,9 @@ class User < ApplicationRecord
   has_many :blockouts, dependent: :destroy
   belongs_to :race, optional: true
   has_and_belongs_to_many :age_ranges
+  has_many :annoucements,
+           dependent:   :restrict_with_error,
```

Comment:
@benjaminwood do you think this is the appropriate dependency?

Reply:
Yeah, that's good.

Reply:
Coolio, thanks for sanity check!